src/utils/parsers/common.ts

```diff
@@ -306,20 +319,25 @@ export abstract class BaseParser {
         return results
     }
 
-    parseFile(file: usage.File): VariableMatch[] {
-        const results: VariableMatch[] = []
-
+    parseFile(file: usage.File): VariableUsageMatch[] {
+        const result: VariableUsageMatch[] = []
+        let fileContent = ''
         for (const line of file.lines) {
-            const matches = this.getAllMatches(line.content)
-            for (const match of matches) {
-                results.push({
-                    fileName: file.name,
-                    line: match.range.start,
-                    name: match.name
-                })
-            }
+           fileContent = fileContent.concat(line.content) 
```

Comment:
in the diff implementation we skip any lines that start with a comment character, do you think something like that could work here too?

Reply:
yeah we can do that too, but i had some work on a different branch handling single line comments